src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPropertySymbol.cs

```diff
@@ -98,46 +97,42 @@ internal sealed class SynthesizedRecordPropertySymbol : SourceOrRecordPropertySy
 
         public override SyntaxList<AttributeListSyntax> AttributeDeclarationSyntaxList => new SyntaxList<AttributeListSyntax>();
 
-        private sealed class GetAccessorSymbol : SynthesizedInstanceMethodSymbol
+        private abstract class AccessorSymbol : SynthesizedInstanceMethodSymbol
         {
-            private readonly SynthesizedRecordPropertySymbol _property;
+            protected readonly SynthesizedRecordPropertySymbol _property;
 
-            public override string Name { get; }
+            public abstract override string Name { get; }
 
-            public GetAccessorSymbol(SynthesizedRecordPropertySymbol property, string paramName)
+            protected AccessorSymbol(SynthesizedRecordPropertySymbol property)
             {
                 _property = property;
-                Name = SourcePropertyAccessorSymbol.GetAccessorName(
-                    paramName,
-                    getNotSet: true,
-                    isWinMdOutput: false /* unused for getters */);
             }
 
-            public override MethodKind MethodKind => MethodKind.PropertyGet;
+            public abstract override MethodKind MethodKind { get; }
 
             public override int Arity => 0;
 
             public override bool IsExtensionMethod => false;
 
-            public override bool HidesBaseMethodsByName => false;
+            public override bool HidesBaseMethodsByName => false; // PROTOTYPE: Should this be true if it hides?
```

Comment:
>// PROTOTYPE: Should this be true if it hides? [](start = 66, length = 46)

This PR is targeting master, no PROTOTYPE comments allowed. Also, I believe C# never hides by name.